src/export/Export.cpp

```diff
@@ -603,9 +507,9 @@ bool Exporter::GetFilename()
          ++i;
          for (int j = 0; j < pPlugin->GetFormatCount(); j++)
          {
-            auto mask = pPlugin->GetMask(j);
-            fileTypes.insert( fileTypes.end(), mask.begin(), mask.end() );
-            if (mPlugins[i]->GetFormat(j) == defaultFormat) {
+            auto formatInfo = pPlugin->GetFormatInfo(j);
+            fileTypes.insert(fileTypes.end(), { formatInfo.mDescription, formatInfo.mExtensions });
```

Comment:
Correctness of this change assumes `SetMask` was nowhere used -- and that is true